demos/tests/run_tests.py

```diff
@@ -238,16 +237,17 @@ def main():
 
         num_failures = 0
 
-        python_module_subdir = "" if platform.system() == "Windows" else "/lib"
         try:
-            pythonpath = "{os.environ['PYTHONPATH']}{os.pathsep}"
+            pythonpath = f"{os.environ['PYTHONPATH']}{os.pathsep}"
         except KeyError:
             pythonpath = ''
         demo_environment = {**os.environ,
             'PYTHONIOENCODING': 'utf-8',
-            'PYTHONPATH': f"{pythonpath}{args.demo_build_dir}{python_module_subdir}",
+            'PYTHONPATH': f"{pythonpath}{args.demo_build_dir}",
```

Comment:
```suggestion
            'PYTHONPATH': f"{pythonpath}{args.demo_build_dir}{os.pathsep}{os.path.join(args.demo_build_dir, 'ctcdecode_numpy')}",
```
why ctcdecode_numpy subdir was removed? https://github.com/openvinotoolkit/open_model_zoo/pull/3345

Reply:
it works without this (at least locally), probably because imported like this: from ctcdecode_numpy import SeqCtcLmDecoder

Reply:
and also work in pre-commit CI